### PR TITLE
Uyuni 2024.01

### PR DESCRIPTION
**Pull request by raulillo82**
Please do not approve till I add the release notes files, as they haven't been merged yet and can still have changes.
But at least the rest can be reviewed.

Edit: release notes already included in one of the commits, ready to be merged during release.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -237,7 +237,7 @@ <h3>Latest news</h3>
               <span>21/12/2023</span>
             </div>
             <div class="news-box-content">
-              <p>Uyuni 2023.12 is now available! Raspberry Pi OS 12 and Amazon Linux 2023 are now supported as clients, CLM filter by package build date and security fixes! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
+              <p>Uyuni 2024.01 is now available! openEuler 22.03 is now supported as a client, reboot required indication for non-Suse distros, one-shot action execution to recurring custom state creation/edition, disabling remote commands execution from web-UI and livenessProbe/readinessProbe added into server container Helm chart! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```
Comment (deneb-alpha) [inline index.html:240]:
```suggestion
              <p>Uyuni 2024.01 is now available! openEuler 22.03 is now supported as a client, reboot required indication for non-SUSE distros and much more! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```

File: pages/devel-version.html
Comment (deneb-alpha) [inline pages/devel-version.html:1]: BTW, on 4.3.11 we  are not submitting Debian 10 that is  way after EOL. 

We  should remove  it  also from here I think. And  actually, it  could  make sense to mention the  EOL in the release notes too

File: pages/stable-version.html
Comment (deneb-alpha) [inline pages/stable-version.html:1]: same as  for  the devel page...
on 4.3.11 we  are not submitting Debian 10 that is  way after EOL. 

We  should remove  it  also from here I think. And  actually, it  could  make sense to mention the  EOL in the release notes too

File: pages/news.html [hunk 2]
```diff
@@ -88,7 +88,8 @@ <h1>NEWS</h1>
 <div class="container page-wrapper">
     <div class="container">
         <div class="row content-padding subpage-content">
-            <p><strong>21/12/2023</strong>Uyuni 2023.12 is now available! Raspberry Pi OS 12 and Amazon Linux 2023 are now supported as clients, CLM filter by package build date and security fixes! Check the <a href='../pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
+	  <p><strong>31/01/2024</strong> - Uyuni 2024.01 is now available! openEuler 22.03 is now supported as a client, reboot required indication for non-Suse distros, one-shot action execution to recurring custom state creation/edition, disabling remote commands execution from web-UI and livenessProbe/readinessProbe added into server container Helm chart! Check the <a href='../pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```
Comment (deneb-alpha) [inline pages/news.html:91]:
```suggestion
	  <p><strong>31/01/2024</strong> - Uyuni 2024.01 is now available! openEuler 22.03 is now supported as a client, reboot required indication for non-SUSE distros and much more! Check the <a href='../pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```

**Review by deneb-alpha — CHANGES_REQUESTED** [review]
submitted a few  suggestions and comments

File: pages/devel-version.html
Comment (raulillo82) [reply pages/devel-version.html:1]: Fixed. OK for the release notes (that I still need to copy here).

File: pages/stable-version.html
Comment (raulillo82) [reply pages/stable-version.html:1]: Fixed. OK for the release notes (that I still need to copy here).

File: pages/devel-version.html
Comment (raulillo82) [reply pages/devel-version.html:1]: Actually no. When writing the release notes, I found out this:
https://wiki.debian.org/LTS -> June 2024. So still supported...
Let's confirm tomorrow, in that case I'll revert the last commit.

File: pages/stable-version.html
Comment (raulillo82) [reply pages/stable-version.html:1]: Actually no. When writing the release notes, I found out this:
https://wiki.debian.org/LTS -> June 2024. So still supported...
Let's confirm tomorrow, in that case I'll revert the last commit.

Comment (raulillo82) [reply pages/stable-version.html:1]: Clarified. It is not about LTS, but Security updates.

File: pages/devel-version.html
Comment (raulillo82) [reply pages/devel-version.html:1]: Clarified. It is not about LTS, but Security updates.

**Review by deneb-alpha — APPROVED** [review]
LGTM

**Review by deneb-alpha — APPROVED** [review]
LGTM

**Review by deneb-alpha — APPROVED** [review]
LGTM

**Pull request merged**